merino-admin/suggestion_providers/models.py

```diff
@@ -0,0 +1,99 @@
+import uuid
+
+from django.db import models
+from polymorphic.models import PolymorphicModel
+
+
+class ProviderConfig(PolymorphicModel):
+    id: models.UUIDField = models.UUIDField(
+        primary_key=True, default=uuid.uuid4, editable=False
+    )
+    exported_name: models.CharField = models.CharField(max_length=256, blank=True, null=True)
+
+    def provider_type(self):
+        return self.get_real_instance().get_type()
+
+    def get_type(self):
+        return '<unknown>'
+
+
+class RemoteSettingsConfig(ProviderConfig):
+    bucket: models.CharField = models.CharField(max_length=256, blank=True, null=True)
```

Comment:
To make things clearer on the admin page, I'd recommend setting default values for these fields and making them unnullable. What do you think?

Reply:
I don't think this is the right.

* The type in Merino for this field is `Option<String>`, so it makes sense to allow it to be left unspecified here.
* `blank=True` means that the user is allowed to leave the field unspecified.
* `null=True` means that when a blank value is encountered, it is stored as a `NULL` in the database.

Some help text in the admin could go a long way to fixing this, but I think if we have a default value we'd have to erase it during serialization, which sounds tricky.

Reply:
My only concern is that when people add/edit entries on the admin page and see those blank fields, they might get confused about whether or not they should fill them out.

I agree with you that we can also leverage the help text (tooltips?) on the page to advertise we have defaults if unspecified. That works for me, too.

Reply:
Here's what it looks like with the help text I added throughout:

![image](https://user-images.githubusercontent.com/305049/149393634-720f12bf-e47b-4b43-8b58-cdf83ad58882.png)